pytorch_lightning/plugins/training_type/rpc_sequential.py

```diff
@@ -87,6 +91,10 @@ def __init__(
             at the same time. Defaults to `True` if
             `get_model_parallel_world_size() > 1`
         """
+        rank_zero_warn(
+            "RPC Sequential Plugin has been deprecated. Please use the `FullShardedPlugin` "
```

Comment:
Needs annotating the deprecation and removal version + actually making it a `DeprecationWarning`